src/Microsoft.Extensions.Logging.Analyzers/Descriptors.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.Extensions.Logging.Analyzers
+{
+    internal class Descriptors
+    {
+        public static DiagnosticDescriptor MEL1NumericsInFormatString = new DiagnosticDescriptor(
+            "MEL1", "Numerics should not be used in logging format string",
+            "Numerics should not be used in logging format string", "Usage", DiagnosticSeverity.Info, true);
+
+        public static DiagnosticDescriptor MEL2ConcatenationInFormatString = new DiagnosticDescriptor(
+            "MEL2", "Logging format string should not be dynamically generated",
+            "Logging format string should not be dynamically generated", "Usage", DiagnosticSeverity.Info, true);
```

Comment:
How does VS display Info level diagnostics? Do you get a squiggly line or a suggestion light bulb? Most analyzers I've used issue warnings for everything.

Reply:
It shows the tiny dots in the beginning:

![image](https://user-images.githubusercontent.com/1697911/30507305-e2bc6efa-9a37-11e7-929d-9c1f9bdb15c2.png)

Reply:
I'm not sure if we want to fail compilation for people who do `WarningAsErrors` because this suggestion is mostly useful when you log a lot.

Reply:
Hmm, too bad that's really short. How hard would it be to set the diagnostic location for the entire formatted string? It makes the dotted underline a little more visible

Reply:
I believe the triple-dots are a fixed size, regardless of how long the location is.

I also think it's still possible to promote info diagnostics to warnings or even errors. So even if we default to Info, those who really want to do things right can promote it as needed.

Reply:
I think that's controlled through https://github.com/dotnet/roslyn/blob/master/docs/compilers/Rule%20Set%20Format.md